### ISSUE TITLE
[1.9] Error "getOnlinePlayers()"

### ISSUE THREAD

**Issue by Androkai** [NLthijs48/AreaShop#127]
Seems like AreaShop is using the obsolete getOnlinePlayers()-method. See this thread for additional information: https://www.spigotmc.org/threads/minecraft-1-9-release.127186/

> The only CraftBukkit change of particular note is the removal of the remapping of legacy < 1.7 API methods. If you have a plugin from 1.6 or older that uses these methods (read: getOnlinePlayers() array), then it may require an update.

Here is the log:
```Java
[02:53:23 WARN]: org.apache.commons.lang.UnhandledException: Plugin AreaShop v2.2.1 generated an exception while executing task 50
        at org.bukkit.craftbukkit.v1_9_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:56)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NoSuchMethodError: org.bukkit.Server.getOnlinePlayers()[Lorg/bukkit/entity/Player;
        at nl.evolutioncoding.areashop.Metrics.postPlugin(Metrics.java:336)
        at nl.evolutioncoding.areashop.Metrics.access$4(Metrics.java:328)
        at nl.evolutioncoding.areashop.Metrics$1.run(Metrics.java:223)
        at org.bukkit.craftbukkit.v1_9_R1.scheduler.CraftTask.run(CraftTask.java:71)
        at org.bukkit.craftbukkit.v1_9_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
        ... 3 more
```

**Comment by frizzbee30** [comment]
It is :) same error.

**Comment by minoneer** [comment]
Getting the same error, looks like a problem with the used Metrics version. I did some digging - @NLthijs48 has already updated metrics after the last release. Which means building the latest version of this repository should fix the issue, however I'm not sure if the current state is stable.

The good news is: running the "fix" command of BuildTools as described [here](https://www.spigotmc.org/threads/minecraft-1-9-release.127186/) (about halfway down the post) fixes the error.

**Comment by NLthijs48** [comment]
I have [updated AreaShop](https://github.com/NLthijs48/AreaShop/releases/tag/v2.2.2) to fix the described errors. Metrics was indeed already fixed in the dev builds, but some tab completions failed because of the getOnlinePlayers() thing. I fixed it in a way that keeps support for 1.7, so at least that should not give problems for people still running that version.
